### PR TITLE
fr_FR: msgmerge + update fuzzy strings

### PR DESCRIPTION
**Pull request by progval**

### CODE REVIEW

File: language/anope.fr_FR.po [hunk 1]
```diff
@@ -9105,5 +9076,66 @@ msgstr "{pseudo | canal}"
 msgid "{nick | channel} memo-text"
 msgstr "{pseudo | canal} texte"
 
+#~ msgid "channel ADD mask"
```
Comment (SadieCat) [inline language/anope.fr_FR.po:9079]: Can you delete the old unused strings from the end of the file?

File: language/anope.fr_FR.po [hunk 2]
```diff
@@ -8869,11 +8840,11 @@ msgid "Your passcode has been re-sent to %s."
...
 
 #, c-format
-msgid "Your password is %s - remember this for later use."
-msgstr "Votre mot de passe est %s - notez-le pour une utilisation ultérieure."
+msgid "Your password is too long. It must be shorter than %u characters."
+msgstr "Votre mot de passe est trop long. Il ne doit pas dépasser %u caractères."
 
 #, c-format
-msgid "Your password is too long. It must not exceed %u characters."
+msgid "Your password is too short. It must must be longer than %u characters."
```
Comment (SadieCat) [inline language/anope.fr_FR.po:8847]: The message here has changed but the translation hasn't. Is that an oversight?

File: language/anope.fr_FR.po (hunk 1 above)
Comment (progval) [reply language/anope.fr_FR.po]: done

File: language/anope.fr_FR.po [hunk 3]
```diff
@@ -6603,6 +6574,9 @@ msgstr ""
 "Définit si le canal donné expirera. Mettre ceci sur ON\n"
 "empêche le canal d'expirer."
 
+msgid "Sets whether the given nickname can be added to a channel access list."
+msgstr ""
```
Comment (SadieCat) [inline language/anope.fr_FR.po:6578]: Empty string?

**Pull request merged**